files/galaxy/tpv/destinations.yml.j2

```diff
@@ -286,9 +286,7 @@ destinations:
       - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.1.2+galaxy0") and helpers.tool_version_lt(tool, "2.3")
         env:
           ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.2"
-        params:
-          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:/data/db/databases/alphafold_databases/2.2:ro"
-      - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.3")
+      - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.3.1+galaxy2")
```

Comment:
Any reason not to stick with `helpers.tool_version_gte(tool, "2.3")`? If you do not, you are excluding versions `2.3.1+galaxy0` and `2.3.1+galaxy1`.

Reply:
That's the purpose cause, based on the tool version, there is a difference in command; the two mentioned versions get DB data from /data path, and the higher versions get the data from /data/2.3 path. That's why I need to bind the database differently.

Reply:
Oh yeah that is true, thanks for clarifying it. Sorry, I should have remembered than even I had to it like this [in the past](https://github.com/kysrpex/usegalaxy-eu-ansible/blob/41161cd7a001fdb94406473bc52253d11fed290a/files/galaxy/tpv/tools.yml#L143-L148).